homeassistant/components/zwave_js/binary_sensor.py

```diff
@@ -0,0 +1,360 @@
+        "enabled": False,
+    },
+    {
+        # NotificationType 8: Power management - Values 10...15
+        # Battery values (mutually exclusive)
+        "type": NOTIFICATION_POWER_MANAGEMENT,
+        "states": [10, 11, 12, 13, 14, 15],
+        "device_class": DEVICE_CLASS_BATTERY,
+        "enabled": False,
+    },
+    {
+        # NotificationType 9: System - State Id's 1, 2, 6, 7
+        "type": NOTIFICATION_SYSTEM,
+        "states": [1, 2, 6, 7],
+        "device_class": DEVICE_CLASS_PROBLEM,
+        "enabled": False,
+    },
+    {
+        # NotificationType 10: Emergency - State Id's 1, 2, 3
+        "type": NOTIFICATION_EMERGENCY,
+        "states": [1, 2, 3],
+        "device_class": DEVICE_CLASS_PROBLEM,
+    },
+    {
+        # NotificationType 11: Clock - State Id's 1, 2
+        "type": NOTIFICATION_CLOCK,
+        "states": [1, 2],
+        "enabled": False,
+    },
+    {
+        # NotificationType 12: Appliance - All State Id's
+        "type": NOTIFICATION_APPLIANCE,
+        "states": list(range(1, 22)),
+    },
+    {
+        # NotificationType 13: Home Health - State Id's 1,2,3,4,5
+        "type": NOTIFICATION_APPLIANCE,
+        "states": [1, 2, 3, 4, 5],
+    },
+    {
+        # NotificationType 14: Siren
+        "type": NOTIFICATION_SIREN,
+        "states": [1],
+        "device_class": DEVICE_CLASS_SOUND,
+    },
+    {
+        # NotificationType 15: Water valve
+        # ignore non-boolean values
+        "type": NOTIFICATION_WATER_VALVE,
+        "states": [3, 4],
+        "device_class": DEVICE_CLASS_PROBLEM,
+    },
+    {
+        # NotificationType 16: Weather
+        "type": NOTIFICATION_WEATHER,
+        "states": [1, 2],
+        "device_class": DEVICE_CLASS_PROBLEM,
+    },
+    {
+        # NotificationType 17: Irrigation
+        # ignore non-boolean values
+        "type": NOTIFICATION_IRRIGATION,
+        "states": [1, 2, 3, 4, 5],
+    },
+    {
+        # NotificationType 18: Gas
+        "type": NOTIFICATION_GAS,
+        "states": [1, 2, 3, 4],
+        "device_class": DEVICE_CLASS_GAS,
+    },
+    {
+        # NotificationType 18: Gas
+        "type": NOTIFICATION_GAS,
+        "states": [6],
+        "device_class": DEVICE_CLASS_PROBLEM,
+    },
+]
+
+
+async def async_setup_entry(
+    hass: HomeAssistant, config_entry: ConfigEntry, async_add_entities: Callable
+) -> None:
+    """Set up Z-Wave binary sensor from config entry."""
+    client: ZwaveClient = hass.data[DOMAIN][config_entry.entry_id][DATA_CLIENT]
+
+    @callback
+    def async_add_binary_sensor(info: ZwaveDiscoveryInfo) -> None:
+        """Add Z-Wave Binary Sensor."""
+        entities: List[ZWaveBaseEntity] = []
+
+        if info.platform_hint == "boolean":
+            entities.append(ZWaveBooleanBinarySensor(client, info))
+        elif info.platform_hint == "notification":
+            entities.append(ZWaveNotificationBinarySensor(client, info))
+        else:
+            LOGGER.warning(
```

Comment:
Is this case possible? We don't see to have this case in the discovery items added.

Reply:
No, it was there for testing discovery scenarios. I'll remove it.

Reply:
done